Cookbook/Technical-Documents/ReleaseProcess.md

```diff
@@ -100,16 +97,15 @@ During this stage, the release manager has the following tasks:
    * Attach the zipped `xcarchive` as an artefact to the GitHub release (if you're using the automated release command, you can find the `*.xcarchive.zip` in the Artifacts top section in the CI build).
 1. Merge `release` branch back to `develop`:
 	* Open the Release PR ( PR from `release` branch targeting `develop`) which has been automatically created.
-  * Resolve the conflicts (if any)
-	* Set as reviewers all the engineers who contributed to the `release` branch, and remove the ones automatically assigned by PullAssigners.
-	* Remove from reviewers list any engineer that has been added by the PullAssigner but haven't contributed to the release branch.
+  * Resolve the conflicts (if any).
+	* In case there are any changes besides updates to app and build versions, then assign as reviewers all the engineers who worked on those changes and remove the ones automatically assigned by PullAssigners. After merging with `develop` the extra changes may be there due to either resolving conflict or because hot fix PR wasn't merged to `develop`.
```

Comment:
```suggestion
	*  In case there are changes other than updates to app and build versions after merging the changes from `develop`, the release engineer should assign as reviewers all the engineers who worked on those changes and remove the ones automatically assigned by PullAssigners. In current workflow every change integrated to release branch is supposed to be go into to `develop` soon after, so differences should be minimal. Any differences might be a result of resolving conflicts or a hot fix PR not being merged previously to `develop`.
```